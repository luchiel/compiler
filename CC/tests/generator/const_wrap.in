void main()
{
int a, b, c;
a = 5 + 8 + 6 - 9;
b = a + 3 * 4;
c = 6 % 4 + (4 << 1);
printf("%d %d %d", a, b, c);
}