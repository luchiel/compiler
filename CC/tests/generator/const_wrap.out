10 22 10